telco-examples/mgmt-cluster/airgap/README.md

```diff
@@ -41,6 +41,21 @@ You need to modify the following folder:
 
 ## Optional modifications
 
+### Add aarch64 architecture support
+
+This is an optional step to add aarch64 architecture support to the management cluster to deploy aarch64 downstream clusters.
+
+1. Modify the helm chart values file for metal3 `kubernetes/helm/values/metal3.yaml` to set the following values:
+
+```yaml
+global:
+  deployArchitecture: arm64
+```
+
+Once you set this value, the management cluster will be able to deploy only aarch64 downstream clusters.
+This is a limitation of the current implementation of the metal3 chart where you can only deploy one architecture at a time.
+NOTE: This limitation will be solved in a future version. 
```

Comment:
```suggestion
NOTE: This is a limitation of the current implementation of the metal3 chart where you can only deploy one architecture at a time. It will be solved in a future version. 
```